app/models/compute_resources/foreman/model/ec2.rb

```diff
@@ -29,7 +25,7 @@ def self.model_name
     end
 
     def capabilities
-      [:image]
+      [:image, :key_pair]
```

Comment:
Thought about that, yet the `:image` is not a part of the mixin - so left it here.